### PR TITLE
Bug hide window

### PR DESCRIPTION
**Pull request by tjmgd**

### CODE REVIEW

File: Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp
Comment (hultonha) [inline Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1078]: question: As this looks like a behavior change it might be something we want UX to signoff on, would it be possible to include a screenshot or video of the before/after behavior please and/or description of exactly what the effect of this change is? Thanks!

File: Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp
Comment (hultonha) [inline Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp:849]: question: Would be great if either @AMZN-AlexOteiza or @jackalbe could take a look at this to verify

**Review by hultonha — COMMENTED** [review]
Change looks good but I have a question to clarify exactly what the change is doing. Thanks!

File: Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp
Comment (tjmgd) [reply Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1078]: This change has been implemented in the latest version of development so it should no longer be relevant. But the PR was submitted before that.

Comment (hultonha) [reply Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1078]: Aaah I see okay, thanks for the context, I'll let @AMZN-AlexOteiza chime in for his take. Cheers!

**Comment by tjmgd** [comment]
Just wondering if there is any update on this?

**Comment by hultonha** [comment]
> Just wondering if there is any update on this?

I'll give @AMZN-AlexOteiza and @jackalbe another poke 🙂 Hopefully should get back to you soon (things start getting a bit quieter at this time of year, hopefully people will still be around to take a look)

**Review by AMZN-AlexOteiza — APPROVED** [review]

File: Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp
Comment (jackalbe) [reply Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp:849]: I just do not see the value of the change, unless I am missing something

**Comment by AMZN-daimini** [comment]
Does this PR include multiple unrelated changes?
Merging latest development into your branch should remove changes that were already submitted and make it clear which ones are under review.

**Comment by tjmgd** [comment]
Merged latest development.

**Review by AMZN-daimini — APPROVED** [review]

**Pull request merged**
